package.json

```diff
@@ -13,8 +13,10 @@
     "pretest" : "semistandard | snazzy"
   },
   "dependencies": {
-    "loader-utils" : "1.1.x",
-    "sharp"        : "0.18.x"
+    "loader-utils" : "1.1.x"
+  },
+  "peerDependencies": {
+    "sharp": "0.x.x"
```

Comment:
This would work assuming we are not using a whitelist, as I proposed above. Otherwise, the list of options is tied to a specific _sharp_ version.